lib/converters/converter-v2-to-v1.js

```diff
@@ -59,6 +59,28 @@ _.extend(Builders.prototype, {
     },
 
     /**
+     * Extracts all requests from a v2 collection or folder
+     *
+     * @param folderOrCollection
+     */
+    extractItems: function (folderOrCollection) {
```

Comment:
Just a thought regarding the public API. extractItems is too generic when clearly the method is focussed on requests only. Maybe rename it to convey intent?